src/gui/folder.cpp

```diff
@@ -191,6 +191,17 @@ QString Folder::path() const
     return p;
 }
 
+bool Folder::ignoreHiddenFiles()
+{
+    bool re(_definition.ignoreHiddenFiles);
+    return re;
```

Comment:
This way it can be checked in the debugger if needed. And I expect the compiler to optimize that as needed.